wraps/python/setup.py

```diff
@@ -0,0 +1,35 @@
+import setuptools
+
+with open("README.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="skender-stock-indicators",
+    version="0.0.1",
+    author="Dave Skender",
```

Comment:
I'll add you here as maintainer [for Python wrapper] if you're okay with that role designation.  From what I can tell from the specs, email is not required; even if it's producing an informational warning.  I've left it out because I'd rather not dox myself - there are other methods to contact us in docs.

Reply:
> there are other methods to contact us in docs.

Yep, you are right. 

> I've added you here as maintainer [for Python wrapper] if you're okay with that role designation. 

Thanks, I love to be there. But could you fix my name to `Dong-Geon Lee`. My name looks not good for readability to native English speaker in my experience. So I usually write my name with `-`.😂